adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala

```diff
@@ -149,8 +144,6 @@ private class FileFilter(private val name: String) extends PathFilter {
   }
 }
 
-import org.bdgenomics.adam.rdd.ADAMContext._
```

Comment:
Nice! I was hoping we could get rid of this.

(On reflection, I don't know why I didn't just try deleting it in the first place!)